.env.example

```
@@ -0,0 +1,61 @@
+# Django Settings
+DEBUG=False
+SECRET_KEY=your-very-secure-secret-key-here
+ALLOWED_HOSTS=localhost,127.0.0.1,django-m.chbk.app
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ALLOWED_HOSTS key should go before the DEBUG key

(UnorderedKey)